client/packages/system/src/Item/api/hooks/useStockItemsWithStats/useStockItemsWithStats.tsx

```diff
@@ -5,7 +5,7 @@ export const useStockItemsWithStats = () => {
   const queryParams = {
     sortBy: { key: 'name', isDesc: false, direction: 'asc' as 'asc' | 'desc' },
     offset: 0,
-    first: 200, // TODO: remove arbitrary limit
+    first: 1000, // TODO: remove arbitrary limit
```

Comment:
should do something about this 🤔

Reply:
Pulling back 2000 manufacturers is working fine for me in HSH. Not sure how many items are needed for this, but I suspect we can increase the limit some more before hitting backend issues. The frontend might struggle with too much data displayed though :(

Reply:
now that I look at it, there's only 57 items currently, so I think we're probably ok with 1000 as a limit for # of stock lines 😁